docs/node/week-3/lesson.md

```diff
@@ -106,31 +121,38 @@ You should complete all of the following sections:
 
 Now when you use
 
-```
+```bash
+# This is an interesting command - Heroku deploys your app by requiring that you push your code to their remote git repository. If you're using the Heroku docs, this remote repo will already be set up, but you will need to add Heroku's git remote repository if you want to use this method on your own private codebase.
 git push heroku main
 ```
 
-After making a `commit` your updated website will be made available online.
+your updated website will be made available online!
+
+Note that `heroku ps:scale web=1` may look like a confusing command, but it's actually broken down as follows:
+
+```bash
+heroku    # the Heroku client you installed through the guide/video
+ps:scale  # telling Heroku that you want to perform a scale operation on a process
+web=1     # in your procfile, you defined a process called web - you're just telling Heroku to scale this process to 1 here
+```
 
 :::
 
 ## 4) City Guide and Paired Programming
 
 :::note Exercise
 
-For this exercises you should split into groups of two or three people.
+For this exercises you should split into groups of two or three people. Focus on just completing level 100 if you can! This exercise is designed to test your pair programming skills and to get you thinking about how to design a full stack application! :) You're not expected to get everything completed.
 
-This challenge should be completed in a Pair Programming style.
+You can either choose to split your focus (such that one person focuses on React and the other focuses on Node) or "swarm" on each side (i.e. both of you working on React or Node at one time - recommendation is to start on the side that at least one of you are not comfortable with!)
```

Comment:
From our chat in-meeting: Let's just suggest swarming, and explicitly tell them _not_ to assign a frontend person and a backend person.